scripts/semconv/templates/SemanticAttributes.ts.j2

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+{%- macro print_value(type, value) -%}
+  {{ "\"" if type == "string"}}{{value}}{{ "\"" if type == "string"}}
+{%- endmacro %}
+{%- macro upFirst(text) -%}
+  {{ text[0]|upper}}{{text[1:] }}
+{%- endmacro %}
+{%- macro lowerFirst(text) -%}
+  {{ text[0]|lower}}{{text[1:] }}
+{%- endmacro %}
+
+// DO NOT EDIT, this is an Auto-generated file from scripts/semconv/templates/{{template}}
+export const {{class}} = {
+  {%- for attribute in attributes if attribute.is_local and not attribute.ref %}
+
+  /**
+  * {% filter escape %}{{attribute.brief | to_doc_brief}}.{% endfilter %}
+    {%- if attribute.note %}
+  *
+  * Note: {% filter escape %}{{attribute.note | to_doc_brief}}.{% endfilter %}
+    {%- endif %}
+    {%- if attribute.deprecated %}
+  *
+  * @deprecated {{attribute.deprecated | to_doc_brief}}.
+    {%- endif %}
+  */
+    {%- if attribute.deprecated %}
+  @Deprecated
```

Comment:
I believe unlike Java there is not direct `@Deprecated` annotation in JS? Won't this be an error when there is some deprecated attars?

Reply:
Ah good point, I will move it to jsdocs: https://jsdoc.app/tags-deprecated.html

Reply:
Removed it